content/news/2022-11-05-bevy-0.9/index.md

```diff
@@ -0,0 +1,1353 @@
+time.set_relative_speed(2.0);
+```
+
+In cases where unscaled values are required, you can use the new "raw" variants of these functions:
+
+```rust
+// The number of seconds elapsed since the last update, with time scaling taken into account.
+let delta = time.delta_seconds();
+
+// The number of seconds elapsed since the last update, with time scaling ignored.
+let raw_delta = time.raw_delta_seconds();
+```
+
+<video controls loop><source  src="time_scale.mp4" type="video/mp4"/></video>
+
+[`Time`]: https://docs.rs/bevy/0.9.0/bevy/time/struct.Time.html
+
+## Time Wrapping
+
+<div class="release-feature-authors">authors: @IceSentry</div>
+
+Some scenarios, [such as shaders](/news/bevy-0-9/#time-shader-globals), need to represent elapsed time values as `f32`, which will suffer from precision issues pretty quickly. To resolve this, [`Time`] has been extended to support "time wrapping":
+
+```rust
+// Wrap once every hour
+time.wrapping_period = Duration::from_secs(60 * 60):
+
+// If one hour and 6 seconds have passed since the app started,
+// this will return 6 seconds.
+let wrapped = time.seconds_since_startup_wrapped_f32();
+```
+
+## What's Next?
+
+Here are some of the things
+
+* **High Level Post Processing Stack**: Now that we have the core post processing pipeline in place, we need to make a higher level system that makes it easier for users to select, configure, and re-order post processing effects on a per-camera basis. Additionally for performance reasons we want to combine as many post processing effects into a single pass as we can, so we need an opinionated set of post processing apis that facilitate this.
+* **More Post Processing Effects**: More anti-aliasing options (TAA, SMAA), more tonemapping algorithm options (Ex: ACES), SSAO
+* **Asset Preprocessing**: We will be investing heavily in our asset pipeline, with a focus on:
+  1. Pre-processing assets to do expensive work "during development time", so Bevy Apps can be deployed with assets that are prettier, smaller, and/or faster to load.
+  2. Enabling configuring assets with .meta files. For example, you could define a texture compression level, the filter it should use, or the target format.
+* **Bevy UI Improvements**: We will continue improving Bevy UI's functionality and expanding its widget library, with a focus on enabling editor experiences.
+* **More Scene Improvements**: Nested scenes, implicit defaults, and inline assets.
+* **Bevy Editor**: We will start prototyping Bevy Editor experiences, starting with scene editor tooling.
+* **Stageless ECS**: Now that the [Stageless RFC](https://github.com/bevyengine/rfcs/pull/45) is merged, we can start implementing stageless scheduling! See the RFC for an outline of the improvements coming. This will be a game changer!
+
+We're also looking for experts in some key areas. Most of our current devs are focused on the efforts above, so if you have interest and experience in the following areas, we would love to hear from you!
```

Comment:
What if we link to issues and PRs marked with `A-Animation` or `A-Audio` here? Also indicate the `#animation-dev` and `#audio-dev` channels on Discord.

Reply:
Done!